### DISCUSSION TITLE
Ability to mark Decoration assets to not collide with tracks, structures, etc.

### DISCUSSION THREAD

**Discussion by EsquimaltNanaimo** [voxeltycoon/issues#515] [Modding]
**Is your feature request related to a problem? Please describe.**
With my signal pack, it can get difficult to place supports and such between tracks at large terminals, or directly next to stations.

**Describe the solution you'd like**
A variable in .decoration files that would make it possible to place them anywhere. This would allow structures, signs and other small objects to be placed between adjacent tracks, for example signal bridge supports in larger yards.

**Describe alternatives you've considered**
n/a

**Additional context**
![signals](https://user-images.githubusercontent.com/34557490/60555976-0452cf80-9cf4-11e9-850c-4711fff43d4e.png)
(thanks refreshfr on the Discord for pointing this out)

**Comment by victorleblais** [comment]
As a temporary work-around, you can set the size of the decoration to 0 in all 3 axis and it will not have a collision box.
